suite/tests/CMakeLists.txt

```diff
@@ -3086,31 +3086,32 @@ if (UNIX)
     tobuild(linux.prctl linux/prctl.c)
   endif ()
   tobuild(linux.mmap linux/mmap.c)
-  tobuild(linux.signal0000 linux/signal0000.c)
-  tobuild(linux.signal0001 linux/signal0001.c)
-  tobuild(linux.signal0010 linux/signal0010.c)
-  tobuild(linux.signal0011 linux/signal0011.c)
-  tobuild(linux.signal0100 linux/signal0100.c)
-  tobuild(linux.signal0101 linux/signal0101.c)
-  tobuild(linux.signal0110 linux/signal0110.c)
-  tobuild(linux.signal0111 linux/signal0111.c)
-  tobuild(linux.signal1000 linux/signal1000.c)
-  tobuild(linux.signal1001 linux/signal1001.c)
-  tobuild(linux.signal1010 linux/signal1010.c)
-  tobuild(linux.signal1011 linux/signal1011.c)
-  tobuild(linux.signal1100 linux/signal1100.c)
-  tobuild(linux.signal1101 linux/signal1101.c)
-  tobuild(linux.signal1110 linux/signal1110.c)
-  tobuild(linux.signal1111 linux/signal1111.c)
-  tobuild(linux.sigplain000 linux/sigplain000.c)
-  tobuild(linux.sigplain001 linux/sigplain001.c)
-  tobuild(linux.sigplain010 linux/sigplain010.c)
-  tobuild(linux.sigplain011 linux/sigplain011.c)
-  tobuild(linux.sigplain100 linux/sigplain100.c)
-  tobuild(linux.sigplain101 linux/sigplain101.c)
-  tobuild(linux.sigplain110 linux/sigplain110.c)
-  tobuild(linux.sigplain111 linux/sigplain111.c)
-  tobuild(linux.sigaction   linux/sigaction.c)
+  tobuild(linux.signal0000     linux/signal0000.c)
+  tobuild(linux.signal0001     linux/signal0001.c)
```

Comment:
Are these all accidental changes?  Best to avoid unrelated changes.

Reply:
They were intentional to align the start of 'linux/'
Still revert?